samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClient.ts

```diff
@@ -8,13 +8,21 @@ import {
     NetworkRequestOptions,
     NetworkResponse
 } from "@azure/msal-common";
+
+/**
+ * After "npx tsc" is executed via the "npm run start" script, Constants.ts and NetworkUtils.ts are compiled to .js and stored in the /dist/utils folder
```

Comment:
In that case, I think we should update the comments when we update the tooling. Additionally, we're importing the compiled js file, not the ts file.

Reply:
Agree with Sameera. This is tech debt which we'd like to avoid. And we also should not be importing anything that is not exported i.e. don't import from individual files.

Reply:
@Robbie-Microsoft I may in general agree that comments should be updated as needed and not future proof, in this case, I do not think importing library compiled samples for constants is a good programming practice. I would prefer to create a local constant and handle this.

Reply:
What do customers do today ? I assume that by referencing MSAL from npm, you don't have this problem? 

IMO, the "samples" from msal repositories are not really "samples", they are "dev apps" used by MSAL developers to test their work. How about

1. Enhance existing official sample (@derisen for thoughts), but this functionality is probably too complex / rarely used to justify the effort
2. Just use documentation (e.g. wiki) 

As for the dev app, agreed on minimizing tech debt.